package.json

```diff
@@ -75,6 +75,9 @@
   "engines": {
     "node": ">=22"
   },
+  "overrides": {
+    "dompurify": "^3.2.4"
+  },
```

Comment:
I believe this was fixed upstream: https://github.com/grafana/grafana/pull/99799

If we just upgrade to `@grafana/data@latest`, does that resolve the warning?